src/mmw/js/src/compare/views.js

```diff
@@ -217,17 +219,27 @@ var InputsView = Marionette.LayoutView.extend({
     ui: {
         chartButton: '#compare-input-button-chart',
         tableButton: '#compare-input-button-table',
+        downloadButton: '#compare-input-button-download'
     },
 
     events: {
         'click @ui.chartButton': 'setChartView',
         'click @ui.tableButton': 'setTableView',
+        'click @ui.downloadButton': 'downloadCSV'
     },
 
     regions: {
         precipitationRegion: '.compare-precipitation',
     },
 
+    modelEvents: {
+        'change:polling': 'toggleDownloadButtonActive'
+    },
+
+    toggleDownloadButtonActive: function() {
+        this.ui.downloadButton.prop('disabled', this.model.get('polling'));
```

Comment:
Fine with this here, but may be worth noting we've generally been doing this in the templates:

https://github.com/WikiWatershed/model-my-watershed/blob/f0c1e0093e25180fc9ab035e973c846b3a7107f1/src/mmw/js/src/data_catalog/templates/pager.html#L9

https://github.com/WikiWatershed/model-my-watershed/blob/1a836bb5f8281b7d7f2b0bedb2a2f7bf9131794a/src/mmw/js/src/modeling/templates/controls/thumbSelect.html#L11

Reply:
Ah that makes sense. I'm going to leave this as is for now, but we can change it if we encounter problems with the jquery version.